src/AasxServerDB/Query.cs

```diff
@@ -0,0 +1,577 @@
+﻿using Extensions;
+using System.Globalization;
+using Microsoft.IdentityModel.Tokens;
+
+namespace AasxServerDB
+{
+    // --------------- Result Schema ---------------
+    public class SmResult

```

Comment:
extract into their own files and organise the wholr query class in its own namespace for better cohesion